.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   test:
-    name: ${{ matrix.task.name}} - ${{ matrix.os.name }} ${{ matrix.python.name }}
+    name: Test - ${{ matrix.os.name }} ${{ matrix.python.name }}
```

Comment:
```suggestion
    name: ${{ matrix.python.name }} ${{ matrix.os.name }} tests
```

Reply:
https://github.com/pytest-dev/pytest-twisted/pull/108 Is my 'plan' to address the OS name (🐧, 🪟, 🍎).  I guess the empty box I get instead of the window character is actually kind of like an ultra simplistic window so maybe I should just merge that.  Python names could be more like the ABI tags (https://www.python.org/dev/peps/pep-0425/#abi-tag).  So, cp27 and pp3 etc.

Should I try a setup like this in an effort to allow all info to be visible even in the short little space provided in the popup you shared?  Rather than shuffling around what you get to see.